.circleci/config.yml

```diff
@@ -184,7 +184,7 @@ jobs:
       - run: pip install tox
       # TODO (#1005): make zsh completion tests pass on macOS machine on CI
       - run: tox -e test-completions -- -vv -k "not zsh"
-      - run: PYTHON_VERSION=3.12-macos  tox -e coverage -- -vv
+      - run: PYTHON_VERSION=3-macos  tox -e coverage -- -vv
```

Comment:
<img width="351" alt="image" src="https://github.com/VirtusLab/git-machete/assets/3383210/2318a163-f1e0-4f64-b184-261ef277a052">

Windows in 3:07, macOS in 2:30, both with coverage. Not bad!